app/views/games/show.html.erb

```diff
@@ -1,3 +1,70 @@
 <h1>Game Show page</h1>
 
-<p>Game name: <%= @game.name %></p>
\ No newline at end of file
+<p>Game name: <%= @game.name %></p> 
+
+<% array = ["A","B","C","D","E","F","G","H"] %> 
+<% unicodefig = ["&#9820;","&#9822;", "&#9821;", "&#9819;", "&#9818;", "&#9821;", "&#9822;", "&#9820;"] %>
+<table id = "ChessBoard"> 
+<% 8.times.each do |number| %> 
+	<% if number% 2 ==0 %>  
+		<tr> 
+			<% if number == 6 %> 
+				<% 8.times.each do |square| %>  
```

Comment:
@lasley so, do I still need to change anything? I am not sure I quite get what you mean with the additional_class variable.